build.gradle

```diff
@@ -52,6 +52,7 @@ subprojects {
         mockitoVersion = '5.12.0'
         truthVersion = '1.4.0'
         tcsVersion = '0.0.1'
+        hiveVersion = '2.3.2'
```

Comment:
Is there a specific reason to use this version of hive-jdbc? 4.0.0 is the more recent version: https://mvnrepository.com/artifact/org.apache.hive/hive-jdbc/4.0.0